user/models.py

```diff
@@ -1,3 +1,21 @@
+from django.contrib.auth.models import AbstractUser
 from django.db import models
 
-# Create your models here.
+
+class User(AbstractUser):
+    username = models.CharField(max_length=256, blank=False, unique=True, )
+    password = models.CharField(max_length=256, blank=False, unique=True, )
```

Comment:
I was mistaken, password hash need not 256 chars, it only needs 128 chars as defined in AbstractBaseUser

password = models.CharField(_("password"), max_length=128)